README.md

```diff
@@ -97,3 +97,15 @@ makim tests.smoke
 - Saransh Chopra - [@Saransh-cpp](https://github.com/Saransh-cpp)
 - Yurely Camacho - [@YurelyCamacho](https://github.com/YurelyCamacho)
 - Ankit Kumar - [@ayeankit](https://github.com/ayeankit)
+
+## Acknowledges
+
+### Python Software Foundation (PSF)
+
+Open Science Labs has been awarded two grants from the [Python Software Foundation (PSF)](https://www.python.org/psf-landing/) in April 2023 and January 2024 to enhance SciCookie. The grants were and will be used to implement best practices recommended by pyOpenSci, improve dependencies and configuration settings, and add new features.
+
+We are grateful for the Python Software Foundation's support. For further details on these grants, we suggest reading some posts on the OSL blog:
```

Comment:
this blog post was just about the first grant. 
maybe you can mention that there is a blog post about the grant received in April, and mention that blog about the second grant will be released in a few months.

additionally you presenting the blog posts as different content. maybe it would be better to present it as the same content but in different languages, maybe something like:
```
We are grateful for the Python Software Foundation's support. For further details on the first grant, received in April 2023, take a look into the blog post `PSF funding open source projects development: SciCookie` here [link], or its version in Spanish here [link].
```
feel free to improve that, it is just an example

Reply:
a friendly reminder about this comment

Reply:
@Anavelyz a friendly reminder about this issue, maybe you can also add now the new blog post as well. just a friendly reminder that these two current blog post listed here are the same, with different version (english and spanish)